packages/react/src/components/ListItem/index.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { Icon } from "..";
+import { ICONS } from "../../constants/";
+
+const ListItem = ({
+  active,
+  title,
+  subtitle,
+  subitem,
+  miniitem,
+  value,
+  leftIcon,
+  rightIcon,
+  hideDivider,
```

Comment:
```suggestion
  withDivider,
```
?

Reply:
I wanted the divider to be on by default. If there's 10 elements, only the last one will be with `hideDivider`

Reply:
You good with that?

Reply:
Yeah, let's keep it this way